scripts/create-draft-release.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console */
+
+const util = require('util')
```

Comment:
is `util` a peerDependency? Because I don't see it in package.json neither in `src/` directory. Maybe use `promisify` from our `helpers/promise` (would require to register Babel, so not 100% sure it's worth)

Reply:
`util` is [part of Node.js](https://nodejs.org/dist/latest-v8.x/docs/api/util.html)... but your comment made me realise I forgot to add the `@octokit/rest` dev dependency to this PR 😅

Reply:
oh nice :D
note to myself: replace our promisify by util.promisify